src/transactions/config/transactions.config.ts

```diff
@@ -0,0 +1,11 @@
+/* istanbul ignore file */
+
+import { registerAs } from '@nestjs/config';
+
+export default registerAs('transactions', () => {
+  const { NOTIFICATIONS_LEGITIMACY_SECRET } = process.env;
```

Comment:
Even though this is needed here, I still prefixed with `NOTIFICATIONS` so it looks more consistent to the person setting them